src/js/common/schemaform/FormApp.jsx

```diff
@@ -65,14 +63,16 @@ export default class FormApp extends React.Component {
 
 
     return (
-      <div className="row">
-        <Element name="topScrollElement"/>
-        <div className="usa-width-two-thirds medium-8 columns">
-          {formConfig.title && !isIntroductionPage && <FormTitle title={formConfig.title} subTitle={formConfig.subTitle}/>}
-          {content}
+      <div>
+        <div className="row">
+          <Element name="topScrollElement"/>
+          <div className="usa-width-two-thirds medium-8 columns">
+            {formConfig.title && !isIntroductionPage && <FormTitle title={formConfig.title} subTitle={formConfig.subTitle}/>}
+            {content}
+          </div>
+          <span className="js-test-location hidden" data-location={trimmedPathname} hidden></span>
         </div>
-        <div className="columns usa-width-one-third medium-4"><AskVAQuestions/></div>
-        <span className="js-test-location hidden" data-location={trimmedPathname} hidden></span>
+        {formConfig.getHelp && formConfig.getHelp(phoneNumber)}
```

Comment:
@cvalarida @jbalboni This is non-idiomatic, and I'd appreciate your thoughts, I realize this is contrary to the initial proposal of only specifying a component in the config (as opposed to the phone number as well), but thought it might be worth exploring.